content/concept-stand-alone-mode.md

```diff
@@ -0,0 +1,103 @@
+
+When you run `rippled` in stand-alone mode, you have to tell it what ledger version to start from:
+
+* Create a [new genesis ledger](#new-genesis-ledger) from scratch.
+* [Load an existing ledger version](#load-saved-ledger) from disk.
+
+**Caution:** In stand-alone mode, you must [manually advance the ledger](#advancing-ledgers-in-stand-alone-mode).
+
+New Genesis Ledger
+-------------------------------------------------------------------------------
+In stand-alone mode, you can have `rippled` create a new genesis ledger. This provides a known state, with none of the ledger history from the production Ripple Consensus Ledger. (This is very useful for unit tests, among other things.)
+
+* To start `rippled` in stand-alone mode with a new genesis ledger, use the `-a` and `--start` options:
+
+```
+rippled -a --start --conf=/path/to/rippled.cfg
+```
+
+In a genesis ledger, the [genesis address](reference-rippled.html#special-addresses) holds all 100 billion XRP. The keys of the genesis address are [hardcoded](https://github.com/ripple/rippled/blob/94ed5b3a53077d815ad0dd65d490c8d37a147361/src/ripple/app/ledger/Ledger.cpp#L184) as follows:
+
+**Address:** `rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh`
+
+**Secret:** `snoPBrXtMeMyMHUVTgbuqAfg1SUTb` ("masterpassphrase")
+
+**Caution:** If you create a new genesis ledger, the hard-coded default [Reserve](concept-reserves.html) is **200 XRP** minimum for funding a new address, with an increment of **50 XRP** per object in the ledger. These values are higher than the current reserve requirements of the production network. (See also: [Fee Voting](concept-fee-voting.html))
+
+
+
+Load Saved Ledger
+-------------------------------------------------------------------------------
+You can start with a ledger version that was saved to disk if your `rippled` server was previously synced with the Ripple peer-to-peer network (either the production network or the [Test Net](tutorial-rippled-setup.html#parallel-networks)).
+
+### 1. Start `rippled` normally. ###
+
+To load an existing ledger, you must first retrieve that ledger from the network. Start `rippled` in networked mode as normal:
+
+```
+rippled --conf=/path/to/rippled.cfg
+```
+
+### 2. Wait until `rippled` is synced. ###
+
+Use the [`server_info` command](reference-rippled.html#server-info) to check the state of your server relative to the network. Your server is synced when the `server_state` value shows any of the following values:
+
+* `full`
+* `proposing`
+* `validating`.
+
+### 3. (Optional) Retrieve specific ledger versions. ###
+
+If you only want the most recent ledger, you can skip this step.
+
+If you want to load a specific historical ledger version, use the [`ledger_request` command](reference-rippled.html#ledger-request) to make `rippled` fetch it. If `rippled` does not already have the ledger version, you may have to run the `ledger_request` command multiple times until it has finished retrieving the ledger.
```

Comment:
I think it's more confusing than helpful to explain why. (Basically, requesting the ledger tends to time out before rippled finishes, and gets distracted with "more important" work related to keeping up with the current ledger.) I think this rippled behavior is obnoxious, but I'm going to leave the documentation as-is for simplicity's sake.